### ISSUE TITLE
Npm start open 2 windows

### ISSUE THREAD

**Issue by DonsWayo** [Mackignacio/electron-angular-quickstart#4]
I don't know if it's an error or it's done that way, but I'd like to know why two windows come out and I can't close one of them have to be open, if I create new components they all start when I run the application?

**Comment by Mackignacio** [comment]
> I don't know if it's an error or it's done that way, but I'd like to know why two windows come out and I can't close one of them have to be open, if I create new components they all start when I run the application?

Yeah its design that way so you can have a better idea on how to handle multiple windows. You can remove it by commenting the createSecondWindow() function inside createWindows().

Please refer to the image below.

![image](https://user-images.githubusercontent.com/12345233/50262655-59f91600-044d-11e9-811c-8fee85660764.png)
